reposcan/database/repository_store.py

```diff
@@ -131,12 +131,24 @@ def delete_content_set(self, content_set_label):
             cur.execute("""select id from repo where content_set_id = %s""", (content_set_id,))
             repo_ids = cur.fetchall()
             for repo_id in repo_ids:
+                cur.execute("select id from module where repo_id = %s", (repo_id,))
+                module_id = cur.fetchone()
```

Comment:
check the cardinality, I think there can be many modules per repo

Reply:
:thinking: sure, I missed it :smile: